core/domain/wipeout_service.py

```diff
@@ -214,6 +229,9 @@ def run_user_deletion_completion(pending_deletion_request):
         user_models.DeletedUserModel(
             id=pending_deletion_request.user_id
         ).put()
+        if pending_deletion_request.normalized_username is not None:
```

Comment:
Why would it ever be None?

(Edit: Ah, I see. See comment below about renaming.)

Reply:
Done.